packages/cli-helpers/src/lib/project.ts

```diff
@@ -33,21 +36,113 @@ export const getInstalledRedwoodVersion = () => {
   }
 }
 
-export const addEnvVarTask = (name: string, value: string, comment: string) => {
+/**
+ * Updates the project's redwood.toml file to include the specified packages plugin
+ *
+ * Uses toml parsing to determine if the plugin is already included in the file and
+ * only adds it if it is not.
+ *
+ * Writes the updated config to the file system by appending strings, not stringify-ing the toml.
+ */
+export const updateTomlConfig = (packageName: string) => {
+  const redwoodTomlPath = getConfigPath()
+
+  const configLines = []
+
+  const configContent = fs.readFileSync(redwoodTomlPath, 'utf-8')
+  const config = toml.parse(configContent)
+
+  if (!config || !config.experimental || !config.experimental.cli) {
+    configLines.push('[experimental.cli]')
+  }
+
+  if (!config?.experimental?.cli?.autoInstall) {
+    configLines.push('  autoInstall = true')
+  }
```

Comment:
If a user explicitly sets this to `false` then this would always fire and switch it for them? They'd end up with 2 lines one enabling and one disabling?